pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -793,9 +793,9 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 			// repetitive log spam
 			switch {
 			case err == images.ErrImagePullBackOff:
-				klog.V(3).Infof("%v start failed: %v: %s", typeName, err, msg)
+				klog.V(3).Infof("%v %+v start failed in pod %v: %v: %s", typeName, spec.container, format.Pod(pod), err, msg)
```

Comment:
It can be something like this. Human readable error first before the long structure describing the image. But again, it is a small suggestion

```suggestion
				klog.V(3).Infof("Image pull failed for %v %+v in pod %v: %s", typeName, spec.container, format.Pod(pod), msg)
```